ocrd/ocrd/decorators.py

```diff
@@ -27,6 +28,13 @@ def _set_root_logger_version(ctx, param, value):    # pylint: disable=unused-arg
                                 default='{}',
                                 callback=lambda ctx, param, value: parse_json_string_or_file(value))
```

Comment:
```suggestion
                                multiple=True,
                                callback=lambda ctx, param, value: parse_json_string_or_file(value))
```

Isn't that still missing?

Reply:
That's in the other params PR https://github.com/OCR-D/core/pull/514/files